seacrowd/sea_datasets/alorese/alorese.py

```diff
@@ -0,0 +1,301 @@
+_LANGUAGES = ["aol", "ind"]
+_LICENSE = Licenses.UNKNOWN.value
+_LOCAL = False
+
+_URLS = _URLS_DICT
+
+_SUPPORTED_TASKS = [
+    Tasks.SPEECH_RECOGNITION,
+    Tasks.MACHINE_TRANSLATION
+] 
+
+_SOURCE_VERSION = "1.0.0"
+_SEACROWD_VERSION = "1.0.0"
+
+class AloreseDataset(datasets.GeneratorBasedBuilder):
+    """ Alorese Corpus is a collection of language data in a couple of Alorese variation (Alor and Pantar Alorese). The collection is available in video, audio, and text formats with genres ranging from Experiment or task, Stimuli, Discourse, and Written materials."""
+
+    SUBSETS = ["t2t", "sptext", "sptext_trans"]
+
+    BUILDER_CONFIGS = [
+            SEACrowdConfig(
+                name=f"{_DATASETNAME}_{subset}_source",
+                version=datasets.Version(_SOURCE_VERSION),
+                description=f"{_DATASETNAME} source schema for {subset} subset",
+                schema="source",
+                subset_id=f"{_DATASETNAME}_{subset}",
+            )
+            for subset in SUBSETS
+        ] + [
+            SEACrowdConfig(
+                name=f"{_DATASETNAME}_t2t_seacrowd_t2t",
+                version=datasets.Version(_SEACROWD_VERSION),
+                description=f"{_DATASETNAME} SEACrowd schema for t2t subset",
+                schema=f"seacrowd_t2t",
+                subset_id=f"{_DATASETNAME}_t2t",
+            ),
+            SEACrowdConfig(
+                name=f"{_DATASETNAME}_sptext_seacrowd_sptext",
+                version=datasets.Version(_SEACROWD_VERSION),
+                description=f"{_DATASETNAME} SEACrowd schema for sptext subset",
+                schema=f"seacrowd_sptext",
+                subset_id=f"{_DATASETNAME}_sptext",
+            ),
+            SEACrowdConfig(
+                name=f"{_DATASETNAME}_sptext_trans_seacrowd_sptext",
+                version=datasets.Version(_SEACROWD_VERSION),
+                description=f"{_DATASETNAME} SEACrowd schema for sptext_trans subset",
+                schema=f"seacrowd_sptext",
+                subset_id=f"{_DATASETNAME}_sptext_trans",
+            ),
+        ]
+
+    DEFAULT_CONFIG_NAME = f"{_DATASETNAME}_source"
+
+    def _info(self) -> datasets.DatasetInfo:
+
+        if self.config.schema == "source":
+            features = datasets.Features(
+                {
+                    "nr": datasets.Value("int64"),
+                    "media_id": datasets.Value("string"),
+                    "audio": datasets.Audio(sampling_rate=16000),
+                    "annotation_aol": datasets.Value("string"),
+                    "annotation_ind": datasets.Value("string"),
+                    "begin_time": datasets.Value("int64"),
+                    "end_time": datasets.Value("int64"),
+                }
+            )
+
+        elif "seacrowd_sptext" in self.config.schema:
+            features = schemas.speech_text_features
+
+        elif self.config.schema == "seacrowd_t2t":
+            features = schemas.text2text_features
+
```

Comment:
I suggest an `else` statement here that raises `ValueError` if the schema isn't one of the presets we have